nodebuilder/p2p/host.go

```diff
@@ -79,9 +91,14 @@ func host(params hostParams) (HostBase, error) {
 		libp2p.DisableRelay(),
 		libp2p.BandwidthReporter(params.Bandwidth),
 		libp2p.ResourceManager(params.ResourceManager),
+		libp2p.ChainOptions(
+			libp2p.Transport(tcp.NewTCPTransport),
+			libp2p.Transport(quic.NewTransport),
+			libp2p.Transport(webtransport.New),
+			wss,
+		),
```

Comment:
Let's still keep using defaults and chain them with wss

Reply:
it can't be replaced: `transports already registered for protocol(s): ws, wss`